humanguide/other-aspects/06-scaling-images.rst

```diff
@@ -0,0 +1,26 @@
+Scaling images
+==============
```

Comment:
This page deals more with sizing than anything else, and there is sizing information elsewhere in this PR. Maybe a section specifically for that is needed.

Reply:
I have re-estructured it a bit as a next section after convergence. It may make more sense now